contracts/interfaces/handlers/IBosonAccountHandler.sol

```diff
@@ -63,15 +63,16 @@ interface IBosonAccountHandler is IBosonAccountEvents {
      * - The dispute resolvers region of protocol is paused
      * - Any address is zero address
      * - Any address is not unique to this dispute resolver
-     * - Number of DisputeResolverFee structs in array exceeds max
-     * - DisputeResolverFee array contains duplicates
      * - EscalationResponsePeriod is invalid
      * - Number of seller ids in _sellerAllowList array exceeds max
      * - Some seller does not exist
      * - Some seller id is duplicated
+     * - DisputeResolver is not active (if active == false)
+     * - Fee amount is a non-zero value. Protocol doesn't yet support fees for dispute resolvers
      *
      * @param _disputeResolver - the fully populated struct with dispute resolver id set to 0x0
-     * @param _disputeResolverFees - array of fees dispute resolver charges per token type. Zero address is native currency. Can be empty.
+    * @param _disputeResolverFees - list of fees dispute resolver charges per token type. Zero address is native currency. See {BosonTypes.DisputeResolverFee}
```

Comment:
This line is missing one whitespace before `*`.

Reply:
fixed